### ISSUE TITLE
Allow configuration for "Enable DNS over HTTPS using:" in gpo

### ISSUE THREAD

**Issue by GHM3434** [mozilla/policy-templates#1074]
Please add an option in group policy settings for the browser setting "Enable DNS over HTTPS using:"

It should let you pick "Default Protection", "Increased Protection" or "Max Protection" <<**_and there should be a field to enter in a custom DNS provider URL._**>>

Reason: ease of enabling encrypted hello to use with dns-over-https for multiple machine environments

**Comment by GHM3434** [comment]
Thank you very much! I see that you fine people added an option to set a custom dns over https provider and it's working great!

There wasn't an option to set "Enable secure DNS using:" to Max Protection setting yet though! I will wait patiently for that.

Thank you again.

**Comment by mkaply** [comment]
We do support Max Protection, although it's not obvious.

If you set Fallback to false, it's Max Protection

https://mozilla.github.io/policy-templates/#dnsoverhttps

This works as of Firefox 124.

**Comment by mkaply** [comment]
> Thank you! So just to confirm if I disable fallback, in my firefox dns over https settings it has "increased protection" selected instead of "max protection". But because fallback is disabled it's the same thing as having max protection selected in FF settings?

Correct, but the UI was supposed to update. I'll see what's going on.

**Comment by GHM3434** [comment]
> > Thank you! So just to confirm if I disable fallback, in my firefox dns over https settings it has "increased protection" selected instead of "max protection". But because fallback is disabled it's the same thing as having max protection selected in FF settings?
> 
> Correct, but the UI was supposed to update. I'll see what's going on.

Thank you!
